tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{38,37,36,35,27}
```

Comment:
Do we not intend to support the removed versions?

Reply:
This PR is dependent on the end of support for python 3.5 and 2.7, as neither of those versions are supported by Pandas, which we require here. For that reason, we can't merge this PR until issue #166 is resolved.

Reply:
There is a classifier in `setup.py` also need to be removed, line 56

Reply:
Also line 17 in `.github/workflows/build.yml`

Reply:
Updated setup.py and build.yml